rotkehlchen/chain/ethereum/modules/liquity/decoder.py

```diff
@@ -195,5 +234,48 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
             LIQUITY_STAKING: (self._decode_lqty_staking_deposits,),
         }
 
+    def _handle_post_decoding(
+            self,
+            transaction: EvmTransaction,
+            decoded_events: list['EvmEvent'],
+            all_logs: list['EvmTxReceiptLog'],
+    ) -> list['EvmEvent']:
+        """
+        Handles post decoding for liuity. It re-applies liquity decoding methods on top of
```

Comment:
typo. Can be changed later

```suggestion
        Handles post decoding for liquity. It re-applies liquity decoding methods on top of
```

Reply:
+1